src/test/kotlin/uk/gov/dluhc/printapi/testsupport/assertj/assertions/TemporaryCertificateAssert.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.dluhc.printapi.testsupport.assertj.assertions
+
+import org.assertj.core.api.AbstractObjectAssert
+import org.assertj.core.util.CheckReturnValue
+import uk.gov.dluhc.printapi.database.entity.TemporaryCertificate
+import java.time.LocalDate
+import java.util.Objects
+
+/**
+ * [TemporaryCertificate] specific assertions - Generated by CustomAssertionGenerator.
```

Comment:
Not a blocker but you could also remove `Generated by CustomAssertionGenerator` in your next PR. Sorry I missed it first time around.